crates/macro/src/lib.rs

```diff
@@ -7,7 +7,8 @@ use quote::quote;
 
 #[proc_macro_attribute]
 pub fn wasm_bindgen(attr: TokenStream, input: TokenStream) -> TokenStream {
-    match wasm_bindgen_macro_support::expand(attr.into(), input.into()) {
+    let attr_args = syn::parse_macro_input!(attr as syn::AttributeArgs);
+    match wasm_bindgen_macro_support::expand(attr_args, input.into()) {
```

Comment:
There may be a way to construct an `AttributeArgs` from a `proc_macro2::TokenStream` but I couldn't get it to work. If that does exist, the `syn` dependency can be removed from this crate.